R/module_teal_with_splash.R

```diff
@@ -95,15 +106,89 @@ srv_teal_with_splash <- function(id, data, modules, filter = teal_slices()) {
           )
         }
       })
-
-      if (!is.reactive(raw_data)) {
-        stop("The delayed loading module has to return a reactive object.")
-      }
       raw_data
     }
 
-    res <- srv_teal(id = "teal", modules = modules, raw_data = raw_data, filter = filter)
-    logger::log_trace("srv_teal_with_splash initialized module with data { toString(get_dataname(data))}.")
+    if (!is.reactive(raw_data)) {
+      stop("The `teal_data_module` has to return a reactive object.", call. = FALSE)
```

Comment:
```suggestion
      stop("`teal_data_module` must return a reactive expression containing a `teal_data` object.", call. = FALSE)
```

I don't remember if I put this in my previous comments.